suite/src/__tests__/correctness/slow/ceramic-cas-integration.test.ts

```diff
@@ -101,20 +89,32 @@ describe('Ceramic<->CAS basic integration', () => {
     expect(doc4.state.log.length).toEqual(2)
 
     // Test document updates
+    // The anchor option is no longer honored when ceramic-one does the anchroing.
     console.log('Updating documents')
-    await doc1.replace(content1, undefined, { anchor: true })
-    await doc2.replace(content1, undefined, { anchor: false })
-    await doc3.replace(content1, undefined, { anchor: false })
-    await doc4.replace(content1, undefined, { anchor: false })
+    await doc1.replace(content1, undefined)
+    await doc2.replace(content1, undefined)
+    await doc3.replace(content1, undefined)
+    await doc4.replace(content1, undefined)
+    await waitForAnchor(doc1)
```

Comment:
this is going to dramatically increase the time this test takes to run.

I think we should just do this one round where we update all 4 streams then wait for them to be anchored, and that's it.  Remove all updates after this point, so that we still have only a single round of anchoring we are waiting on in this test.

We would then lose any integration test coverage that anchoring works when multiple consecutive writes are anchored by a single time event, but it sounds like we have no reliable way to introduce such a scenario anymore.

Reply:
I started with that plan however this creates races the test cannot deal with. The current conflict resolution rules prefer branches that have the earlier time event. Its easy to get unanchored writes pruned in these tests because we anchor so quickly.

Basically what happens is that the test writes 4 events to a stream. If its right before an anchor batch then say only the first two writes get anchored. But the test didn't wait to see if the updates are anchored and so the last two writes are pruned and it causes the test to fail.

This is basically the race we already know exists but in these test envs the race window is large. 

As discussed before the solution here is to change/update conflict resolution rules and make time events branches so that time events cannot prune valid data events. Until we have that fix we have to wait for anchors in the test case.

Reply:
sorry, my proposal was to do a single update to each stream and get them anchored.  Not to do 4 updates in a row

Reply:
Ok can do, I misunderstood. Keeping it simple works for me.